CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to Forgerock's Javascript SDK
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to Forgerock and its packages, which are hosted in the [Forgerock](https://github.com/forgerock) on GitHub.
+
+### Git Flow
+
+Please fork the repository to your own github account. From there, you can add the forgerock repository as an upstream.
+
+`git remote add upstream git@github.com:ForgeRock/forgerock-javascript-sdk.git`
+
+Here is a checklist of items to ensure your Pull Request is reviewed and ready to be merged:
+
+- [ ] Please make all pull requests against `develop` branch.
+- [ ] Fill out the Pull Request template
+- [ ] When your PR is ready to merge please rebase your commit into 1 commit.
+- [ ] Commits should be verified (meaning you cannot squash your commit on github).
+- [ ] Please ensure your code is passing CI in order to get a review
+- [ ] Structure your commit messages similarly to the angular commit guideleines
```

Comment:
Please capitalize Angular and provide a link to the reference.

Reply:
Please add periods '.' to the end of the sentences.